packages/kbn-utility-types/test-d/union_to_intersection.ts

```diff
@@ -17,12 +17,12 @@
  * under the License.
  */
 
-import { expectType } from 'tsd';
```

Comment:
Note to reviewers: All the changes in these files from `expectType` to `expectAssignable` are because of this breaking change to `tsd`: https://github.com/SamVerschueren/tsd/commit/e4c78c77be8bfa78268d2155a6f46bf43490c4f9